examples/README.md

```diff
@@ -0,0 +1,61 @@
+# Examples for Testing Features Locally
+
+The example projects are meant to be used to test features locally by contributors working on this SDK. In the `requirements.txt` file, the deepgram package used is the local version:
+
+```
+-e ../../
+```
+
+## Steps to Test Your Code
+
+If you are contributing changes to this SDK, you can test those changes by using the `prerecorded` or `streaming` projects in the `examples` folder. Here are the steps to follow:
+
+### Add Your Code
+Make your changes to the SDK (be sure you are on a branch you have created to do this work).
+
+### Install dependencies
+
+Run the following command to install the project dependencies:
+
+`pip install -r requirements.txt`
```

Comment:
Does the user have to be in a specific repo to run this command?

Reply:
Yes, in the examples folder in either the prerecorded or streaming project. I'll add that clarification.